pkg/hubble/parser/threefour/parser.go

```diff
@@ -36,6 +37,7 @@ type Parser struct {
 	linkGetter     getters.LinkGetter
 
 	epResolver *common.EndpointResolver
+	enabled    bool
```

Comment:
Rename to something more verbose like `enrichL3L4Policy`.

Reply:
Done